src/gpu/d3d12/SDL_gpu_d3d12.c

```diff
@@ -3888,9 +3888,6 @@ static void D3D12_INTERNAL_TrackUniformBuffer(
         commandBuffer->usedUniformBuffers = (D3D12UniformBuffer **)SDL_realloc(
             commandBuffer->usedUniformBuffers,
             commandBuffer->usedUniformBufferCapacity * sizeof(D3D12UniformBuffer *));
-        for (i = commandBuffer->usedUniformBufferCount; i < commandBuffer->usedUniformBufferCapacity; i += 1) {
```

Comment:
What's going on here?

Reply:
This was part of the zeroing change set, the zerop'd pointer is uninitialized and zerop causes it to be de-referenced which causes a crash. There's no need to zero it anyway.